packages/react/package.json

```diff
@@ -4,10 +4,12 @@
   "scripts": {
     "start": "npm-run-all core:build dev:start",
     "dev:start": "yarn run clean && parcel index.html --cache-dir=./.parcel-cache",
-    "build": "npm-run-all -s build:ts build:package build:copy",
+    "build": "npm-run-all -s build:ts build:package build:copy build:distribute",
+    "build:distribute": "cd ../../scripts/ && node ./package-distribute.js @clr/react",
```

Comment:
You couldn't directly call the script without changing directories?
```
node ../../scripts/package-distribute.js @clr/react
```

Reply:
I think I can with proper use of the path resolve I just havent had time to finish this up yet